PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -206,10 +206,17 @@ class HTTPClientTests: XCTestCase {
         }
 
         self.client.performRequest("GET", path: path, body: nil, headers: nil) { (status, data, responseError) in
-            successFailed = (status >= 500) && (data == nil) && (error == responseError as NSError?)
+            if let responseNSError = responseError as? NSError {
+                successFailed = (status >= 500
+                                 && data == nil
+                                 && error.domain == responseNSError.domain
+                                 && error.code == responseNSError.code)
```

Comment:
this had been bugging me for a while - apparently on iOS 14, `error != responseError as NSError?`. 
So this would fail. I'm just manually checking the domain and code, which should suffice for the purposes of the test.